docs/product/components/progress-bars.html

```diff
@@ -231,6 +231,99 @@
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+
+        <div class="s-progress s-progress__segmented fc-orange-400" style="--progress-value: 1; --progress-value-max: 4;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+
+        <div class="s-progress s-progress__segmented fc-green-400" style="--progress-value: 7; --progress-value-max: 10;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+
+        <div class="s-progress s-progress__segmented fc-red-600" style="--progress-value: 18; --progress-value-max: 20;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+    </div>
+</div>
+
+    <p class="stacks-copy">By default, the segmented progress bars are 256px wide. These dimensions can be modified by adding
+        t-shirt sizing classes to <code class="stacks-code">.s-progress__segmented</code>: <code
+            class="stacks-code">.s-progress__sm</code>, <code class="stacks-code">.s-progress__md</code>, and <code
+            class="stacks-code">.s-progress__lg</code></p>
+
+<div class="stacks-preview">
+{% highlight html %}
+<div class="s-progress s-progress__segmented s-progress__sm" style="--progress-value-max: 4; --progress-value: 1;">
+    <svg viewBox="0 0 100 3">
+        <line x1="1" y1="1" x2="100" y2="1" />
+        <line x1="1" y1="1" x2="100" y2="1" />
+    </svg>
+</div>
+{% endhighlight %}
+    <div class="stacks-preview--example">
+        <div class="s-progress s-progress__segmented s-progress__sm" style="--progress-value-max: 4; --progress-value: 1;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+
+        <div class="s-progress s-progress__segmented" style="--progress-value-max: 4; --progress-value: 1;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+
+        <div class="s-progress s-progress__segmented s-progress__md" style="--progress-value-max: 4; --progress-value: 1;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
+                <line x1="1" y1="1" x2="100" y2="1" />
+            </svg>
+        </div>
+
+        <div class="s-progress s-progress__segmented s-progress__lg" style="--progress-value-max: 4; --progress-value: 1;">
+            <svg viewBox="0 0 100 3">
+                <line x1="1" y1="1" x2="100" y2="1" />
```

Comment:
The thing that's weird to me about these being built off of SVG is that they scale proportionately, unlike our other line-based progress bars. Is there a way to build these with actual markup and flex? Perhaps we build them with `grid` and reach for `gap`?

Another unfortunate side-effect of the SVG is that they're blurry on 1x screens like mine 🤔

Reply:
I'm pretty sure we could do it with an HTML grid. I was trying to avoid that because I think it means we'll have x number of divs nested inside... 1 for each segment. Any idea on how to do it with having potentially 20 extra divs?